docs/2.3/user-manual.md

```diff
@@ -113,6 +114,28 @@ total 8.0K
 -rw------- 1 joe staff 1.5K Aug 10 16:16 joe-cert.pub
 ```
 
+### SSH Certificates for Automation
+
+Regular users of Teleport must request an auto-expiring SSH certificate,
+usually every day. This doesn't work for non-intractive scripts, like cron jobs
+or CI/CD pipeline.
+
+It is recommended to create a separate Teleport user for such bots and request
+a certificate for them with a long time to live (TTL). In this example we're
+creating a certificate with a TTL of 10 years for the Jenkins user and storing
```

Comment:
why `the` here?

Reply:
For such automation, it is recommended to create a separate Teleport user for bots and request a certificate for them with a long time to live (TTL).